### ISSUE TITLE
Video progress bar is not accessible using keyboard.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1374]
**Actual Result:**
Video progress bar is not accessible using keyboard.

**Note:**
Controls present under settings are not accessible using narrator.

**Expected Result:**
Video progress bar should be accessible using keyboard.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/ ](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Navigate to 'Show a video attachment from Vimeo' button and activate it.
3. Video will appear at the bottom of the page.
4. Navigate the video using tab key and try to reach at video progress bar.
5. Observe if video progress bar is accessible using keyboard or not.

**User Impact:**
If any control is not accessible using keyboard then the user who use keyboard to navigate the page will not able to access that control.

**MAS Reference:**
[MAS 2.1.1 - Keyboard (36)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachments:**
[MAS2.1.1_Keyboard.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2594604/MAS2.1.1_Keyboard.zip)

**Comment by compulim** [comment]
This is not a bug from Web Chat. The video content is an IFRAME owned by Vimeo.

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed with Accessibility team as External

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG2.1.1
